src/auth.py

```diff
@@ -199,15 +199,22 @@ def authenticate(f):
     @wraps(f)
     async def decorated_function(*args, **kwargs):
         try:
-            await get_user_id()
+            user_id = await get_user_id()
         except Exception as e:
             logger.error(f"Failed to authenticate user: {e}")
             raise Unauthorized(str(e))
```

Comment:
```suggestion
            raise Unauthorized()
```
This is the case where I'd avoid giving user any info back other than a pure 401.